static/src/stylesheets/module/email-signup/_form.scss

```diff
@@ -155,7 +155,7 @@
 .email-sub__message__headline,
 .email-sub__message__description {
     @include fs-textSans(3);
-    color: #000000;
+    color: #ffffff;
```

Comment:
You're probably going to end up having the same problem in other places the email is used, so I would leave this as is and add the color override scoped to the Footer Mods section https://github.com/guardian/frontend/blob/ma/subscribe-text-fix/static/src/stylesheets/module/email-signup/_form.scss#L196